sdk/translation/azure-ai-translation-text/README.md

```diff
@@ -55,16 +57,52 @@ az cognitiveservices account keys list --resource-group <your-resource-group-nam
 
 #### Create a `TextTranslationClient` using an API key and Region credential
 
-Once you have the value for the API key and Region, create an `TranslatorCredential`. This will allow you to
+Once you have the value for the API key and Region, create a `TranslatorCredential`. This will allow you to
 update the API key without creating a new client.
 
 With the value of the `endpoint`, `credential` and a `region`, you can create the [TextTranslationClient][client_sample]:
 
 <!-- SNIPPET: sample_text_translation_client.create_text_translation_client_with_credential -->
 
 ```python
+from azure.ai.translation.text import TextTranslationClient, TranslatorCredential
+
+endpoint = os.environ["AZURE_TEXT_TRANSLATION_ENDPOINT"]
+apikey = os.environ["AZURE_TEXT_TRANSLATION_APIKEY"]
+region = os.environ["AZURE_TEXT_TRANSLATION_REGION"]
+
 credential = TranslatorCredential(apikey, region)
-text_translator = TextTranslationClient(credential, endpoint)
+text_translator = TextTranslationClient(endpoint=endpoint, credential=credential)
+```
+
+<!-- END SNIPPET -->
+
+#### Create a `TextTranslationClient` using Entra (Azure Active Directory) Authentication
+
+`AzureKeyCredential` authentication is used in most of the examples, but you can also authenticate with Entra (Azure Active Directory) using the [Azure Identity library][azure_identity].
+
+Create a [custom subdomain][custom_subdomain] for your resource in order to use this type of authentication.  Use this value for the `endpoint` variable for `AZURE_TEXT_TRANSLATION_ENDPOINT`.  Only custom subdomains are supported.
```

Comment:
> Only custom subdomains are supported.

Please make sure that it is clear that we mean supported in Python SDK. Without SDK you can use AAD auth for MT endpoints.

Reply:
Updated.